diff --git a/angr/sim_state.py b/angr/sim_state.py
index 4d6bff5..ee83335 100644
--- a/angr/sim_state.py
+++ b/angr/sim_state.py
@@ -657,7 +657,7 @@ class SimState(PluginHub):
                 None
             )
             if plugin_common_ancestor is None and \
-                    plugin_class is SimStateHistory and \
+                    (plugin_class is SimStateHistory or plugin_class.__name__ == "SymbolicMemory") and \
                     common_ancestor_history is not None:
                 plugin_common_ancestor = common_ancestor_history

--
2.7.4
